docs/services/navigation/_index.md

```diff
@@ -118,6 +118,10 @@ The following attributes are available for `Navigation` services:
 | `base` | string | **Required** | The `name` you have configured for the [base](/components/base/) you are operating with this service. |
 | `movement_sensor` | string | **Required** | The `name` of the [movement sensor](/components/movement-sensor/) you have configured for the base you are operating with this service. |
 | `motion_service` | string | Optional | The `name` of the [motion service](/services/motion/) you have configured for the base you are operating with this service. If you have not added a motion service to your robot, the default motion service will be used. Reference this default service in your code with the name `"builtin"`. |
+| `vision_services` | array | Optional | The `name` of each [vision service](/services/motion/) you have configured for the base you are operating with this service. |
+| `position_polling_frequency` | float | Optional | The frequency to poll for the position of the robot. <br> Default: `2` |
+| `obstacle_polling_frequency_hz` | float | Optional | The frequency in hz to poll each vision service for new obstacles. <br> Default: `2` |
+| `plan_deviation_m` | float | Optional | The distance in meters that a robot is allowed to deviate from the motion plan. <br> Default: `1e9`|
```

Comment:
`1e9` was an arbitrarily high number I chose to effectively turn off this for testing purposes (right now this is larger than the width of the earth).  It should be changed pretty soon within the next week.  I think lets go with `3` for now

Reply:
Sounds good, changed!!